packages/core-bridge/index.d.ts

```diff
@@ -210,9 +220,19 @@ export interface TelemetryOptions {
    *
    * Turned off by default
    */
-  metrics?: MetricsExporter;
+  metrics?: RequireExactlyOne<
```

Comment:
I don't see `RequireExactlyOne` defined or imported here.

Reply:
Can we keep the types a bit simpler?
It'll be very hard to understand the shape of these types with all of these transformations.

Reply:
We also require either prom or otel and temporality is optional, so the name RequireExactlyOnce is confusing here.

Reply:
Removed usage RequiredExactlyOnce